core/templates/dev/head/pages/exploration_editor/editor_tab/StateSolution.js

```diff
@@ -51,7 +51,7 @@ oppia.controller('StateSolution', [
       ExplorationHtmlFormatterService.getInteractionHtml(
         stateInteractionIdService.savedMemento,
         explorationStatesService.getInteractionCustomizationArgsMemento(
-          EditorStateService.getActiveStateName()),
+          EditorStateService.getActiveStateName()), true,
```

Comment:
Ditto, move true to next line. Ditto below as well.

Reply:
Done